looper/command_models/commands.py

```diff
@@ -0,0 +1,91 @@
+"""
+`pydantic` models for `looper` commands and a wrapper class.
+"""
+
+from dataclasses import dataclass
+from typing import Optional
+
+import pydantic
+
+from ..const import MESSAGE_BY_SUBCOMMAND
+from .arguments import Argument, ArgumentEnum
+
+
+@dataclass
+class Command:
+    """
+    Representation of a command
+
+    :param str name: command name
+    :param str description: command description
+    :param list[Argument] arguments: list of arguments supported by this command
+    """
+
+    name: str
+    description: str
+    arguments: list[Argument]
```

Comment:
Why not also import `List` from typing and make this `List[Argument]`?

Reply:
Since Python 3.9, `list` are generic types and are preferred over their `typing` counterparts. Took that phrase from this SO answer: https://stackoverflow.com/a/62033243
But then again, from `setup.py`, it seems like `looper` is supposed to be Python 3.8-compatible. So I'd be happy to change this, if we want to insist on that compatibility.

Reply:
TIL, thanks! sounds good :)